framework/i18n/Formatter.php

```diff
@@ -826,28 +827,30 @@ public function asRelativeTime($value, $referenceTime = null)
             if ($interval->s == 0) {
                 return Yii::t('yii', 'just now', [], $this->locale);
             }
+
             return Yii::t('yii', 'in {delta, plural, =1{a second} other{# seconds}}', ['delta' => $interval->s], $this->locale);
-        } else {
```

Comment:
It would be easier to understand if we extract these blocks to separate methods.